src/components/timeline/package.json

```diff
@@ -1,20 +1,19 @@
 {
-  "name": "@ecl/vanilla-component-timeline",
+  "name": "@ecl/timeline",
   "author": "European Commission",
-  "main": "timeline.js",
-  "module": "timeline.js",
-  "style": "timeline.scss",
-  "sass": "timeline.scss",
   "license": "EUPL-1.2",
   "version": "4.9.0",
   "description": "ECL Timeline",
-  "dependencies": {
-    "@ecl/dom-utils": "4.9.0",
-    "@ecl/vanilla-layout-grid": "4.9.0"
-  },
   "publishConfig": {
     "access": "public"
   },
+  "main": "timeline.js",
+  "module": "timeline.js",
+  "style": "timeline.scss",
+  "dependencies": {
+    "@ecl/dom-utils": "4.9.0",
+    "@ecl/button": "4.9.0"
```

Comment:
if the vanilla component had @ecl/vanilla-layout-grid as a dependency now we should still have it but as @ecl/grid